source/agora/test/Base.d

```diff
@@ -907,7 +907,103 @@ public class TestAPIManager
             ex.line = line;
             throw ex;
         }
+        retryFor(nodes[0].client.getBlocksFrom(0, 1).length > 0, 2.seconds);
     }
+
+    /*******************************************************************************
+
+        Add blocks up to the provided height
+
+        Params:
+            target_height = the target height after adding blocks.
+            enrolled_height = the height with the active enrollments
```

Comment:
I'm not sure I understand the meaning of this parameter. "the active enrollments => which active enrollments? New enrollments? It should state what is the effect of setting this parameter.

Reply:
I added some documentation for this